packages/core/src/forms/control/control.element.scss

```diff
@@ -71,7 +71,7 @@ $fallback-width: calc(#{$cds-global-layout-space-xxl} * 2);
   cursor: default;
 }
 
-cds-control-action.status {
+[cds-button-action].status {
```

Comment:
All the action buttons have a `[cds-button-action]` attribute that way you can query/slot any action button or alias (`cds-button-action` or `cds-control-action`). The datagrid has some similar queries since you can project a regular action button or one of the other types. I can add a comment to this line here as well https://github.com/coryrylan/clarity/blob/topic/core-action-button/packages/core/src/button-action/button-action.element.ts#L43

Reply:
I'm not a fan of the name however, if I think of something I'll update it. Could technically could do something like `[cds-button]` on the base then components could query for any `[cds-button]` attr which basically all clarity buttons would then have.